start

```diff
@@ -274,7 +274,7 @@ function init_stellar_core() {
 
   run_silent "finalize-core-config-pgpass" sed -ri "s/__PGPASS__/$PGPASS/g" etc/stellar-core.cfg
   local RUN_STANDALONE=false
-  if [ "$NETWORK" = "standalone" ] && [ "$HORIZON_ENABLE_CAPTIVE_CORE" = "false" ]; then
+  if [ "$NETWORK" = "standalone" ] && [ "$ENABLE_HORIZON_CAPTIVE_CORE" = "false" ]; then
```

Comment:
@acharb Since by fixing this bug you're adding a new case where captive core will be used, can you add a test case to the build matrix so that we test each build with captive core, and make sure the standalone network works with this change?

You can add a new parameter to the matrix here:
https://github.com/stellar/docker-stellar-core-horizon/blob/a3ebbae7339875450a7316928d4d0960c778fd80/.github/workflows/test.yml#L10-L14